service/frontend/workflow_handler.go

```diff
@@ -3727,6 +3727,21 @@ func (wh *WorkflowHandler) ListBatchOperations(
 	}, nil
 }
 
+// PollNexusTaskQueue implements Handler.
+func (*WorkflowHandler) PollNexusTaskQueue(context.Context, *workflowservice.PollNexusTaskQueueRequest) (*workflowservice.PollNexusTaskQueueResponse, error) {
+	panic("unimplemented")
```

Comment:
please
```suggestion
	return nil, serviceerror.NewUnimplemented("unimplemented")
```

Reply:
Don't worry about this, I'm going to implement this in a follow up PR and this is not being merged into `main` yet.